### PR TITLE
SEO for Adaptors

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR do SEO for Adaptors

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

image
image
image

### CODE REVIEW

**Comment by zuobiao-zhou** [comment]
PTAL @Anonymitaet TIA.

**Review by Anonymitaet — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]
Thank you!

File: docs/adaptors-storm.md [hunk 1]
```diff
@@ -10,7 +11,7 @@ An application can inject data into a Storm topology via a generic Pulsar spout,
 
 ## Using the Pulsar Storm Adaptor
 
-Include dependency for Pulsar Storm Adaptor:
+To use the Pulsar Storm Adaptor, you need to include dependency for Pulsar Storm Adaptor:
 
 ```xml
 <dependency>
```
Comment (tisonkun) [inline docs/adaptors-storm.md:17]: Out of thread, @cbornet is the GAV of pulsar storm adaptor changed?

Comment (cbornet) [reply docs/adaptors-storm.md:17]: What is "GAV" ?

Comment (tisonkun) [reply docs/adaptors-storm.md:17]: group id, artifact id, version; that is, a unique identifier for a dependency.

Comment (cbornet) [reply docs/adaptors-storm.md:17]: Oh ok. The storm adapter has moved to the Storm project so we should update the doc.

**Pull request merged**